docs/quality_assurance.md

```diff
@@ -41,26 +41,26 @@ IBC packets:
 | Concern | Code Review | Unit Testing | Diff. testing | Testnet | Protocol audit |
 | ------- | ----------- | ------------ | ------------- | ------- | ----- |
-| Create IBC clients | `TODO` (ibc-go team) | `Done` | `In future` | `TODO` | `NA` |
-| Getting consumer `UnbondingPeriod` from IBC client | `TODO` (ibc-go team) | `??` | `Not planned` | `TODO` | `NA` |
-| Create CCV channel (handshake) | `TODO` (ibc-go team) | `Done` | `In future` | `TODO` | `TODO` |
-| Sending IBC packets <br /> - see `x/ccv/utils/utils.go:SendIBCPacket()` | `TODO` (ibc-go team) | `Done` | `Done` | `TODO` | `NA` |
-| Handling acknowledgments | `TODO` (ibc-go team) | `Low coverage` | `Scheduled` | `TODO` | `NA` |
-| Handling timeouts | `TODO` (ibc-go team) | `Low coverage` | `In future` | `TODO` | `NA` |
-| Handling IBC client expiration | `TODO` (ibc-go team) | `??` | `In future` | `TODO` | `NA` |
-| ICS-20 channel creation | `TODO` (ibc-go team) | `??` | `In future` | `TODO` | `NA` |
-| ICS-20 transfer | `TODO` (ibc-go team) | `??` | `Not planned` | `TODO` | `NA` |
-| Changes in IBC-GO testing suite | `TODO` (ibc-go team) | `??` | `Partially done` | `TODO` | `NA` |
+| Create IBC clients | `Scheduled` (ibc-go team) | `Done` | `Future work` | `Scheduled` | `NA` |
+| Getting consumer `UnbondingPeriod` from IBC client | `Scheduled` (ibc-go team) | `??` | `NA` | `NA` | `NA` |
+| Create CCV channel (handshake) | `Scheduled` (ibc-go team) | `Done` | `Future work` | `Scheduled` | `Scheduled` |
+| Sending IBC packets <br /> - see `x/ccv/utils/utils.go:SendIBCPacket()` | `Scheduled` (ibc-go team) | `Done` | `Done` | `Scheduled` | `NA` |
+| Handling acknowledgments | `Scheduled` (ibc-go team) | `Low coverage` | `Scheduled` | `Scheduled` | `NA` |
+| Handling timeouts | `Scheduled` (ibc-go team) | `Low coverage` | `Future work` | `Scheduled` | `NA` |
+| Handling IBC client expiration | `Scheduled` (ibc-go team) | `??` | `Future work` | `Scheduled` | `NA` |
+| ICS-20 channel creation | `Scheduled` (ibc-go team) | `??` | `Future work` | `Scheduled` | `NA` |
+| ICS-20 transfer | `Scheduled` (ibc-go team) | `??` | `NA` | `Scheduled` | `NA` |
+| Changes in IBC-GO testing suite | `Scheduled` (ibc-go team) | `NA` | `WIP` | `NA` | `NA` |
 
 ## Integration with Cosmos SDK
 
 A prerequisite of the code review is to open a PR with all the [SDK changes](https://github.com/cosmos/cosmos-sdk/tree/interchain-security-rebase) needed by Interchain Security.
 
 | Concern | Code Review | Unit Testing | Diff. testing | Testnet | Protocol audit |
 | ------- | ----------- | ------------ | ------------- | ------- | ----- |
-| Changes to staking module | `TODO` (sdk team) | `Low coverage` | `Partially done` | `TODO` | `NA` |
-| Changes to slashing module | `TODO` (sdk team) | `??` | `Not planned` | `TODO` | `NA` |
-| Changes to evidence module | `TODO` (sdk team) | `??` | `Not planned` | `TODO` | `NA` |
+| Changes to staking module | `Scheduled` (sdk team) | `Low coverage` | `WIP` | `Scheduled` | `NA` |
```

Comment:
Hey, so in this particular instance where `Partially done` -> `WIP` I really meant something like: it partially addresses the concern but it is done. For example the diff testing _does_ test many parts of the staking module changes but maybe not all, and I dont' currently have plans to extend that.

Reply:
Same for all places where I wrote `Partially done`.

Reply:
Yeah sure, that's better!